src/diffusers/models/auto_model.py

```diff
@@ -16,9 +16,12 @@
 import os
 from typing import Optional, Union
 
+from huggingface_hub import constants, hf_hub_download
 from huggingface_hub.utils import validate_hf_hub_args
 
+from .. import pipelines
```

Comment:
I think if we move this import inside the `try` block, we should be able to get rid of the circular import problem: https://github.com/huggingface/diffusers/actions/runs/14787632114/job/41518909088?pr=11401#step:15:68